halo2-lib-js/src/halo2lib/CircuitValue256.ts

```diff
@@ -0,0 +1,61 @@
+import { Halo2LibWasm } from "@axiom-crypto/halo2-wasm/web";
+import { CircuitValue } from "./CircuitValue";
+import { convertInput } from "../shared/utils";
+
+export class CircuitValue256 {
+    private _value: bigint;
+    private _circuitValue: [CircuitValue, CircuitValue];
+    private _halo2Lib: Halo2LibWasm;
+
+    constructor(_halo2Lib: Halo2LibWasm, { value, hi, lo }: { value?: bigint | string | number, hi?: CircuitValue, lo?: CircuitValue }) {
+        this._halo2Lib = _halo2Lib;
```

Comment:
Let's add a check that `value` is not negative. Does typescript have a `biguint` type?